magento/pwa-studio
# #1222 Filters (Layered Navigation) - Add keyboard focus and adjust the way tab order should work

## Description

This PR adds the a11y changes to the Layered Navigation (Filters) modal on the category pages according to the specifications in #1222 .

## Related Issue

Closes #1222.

## Acceptance



### Verification Stakeholders

@dpatil-magento @sirugh 

### Specification



### Verification Steps


1. Go to any category page that contains products and has filter items.
2. Open the Filters modal.
3. Make sure Filters modal opens.
4. Make sure the modal's close (x) icon is focused.
5. From the keyboard, press the `Tab` key.
6. Make sure the focus switches to the next element. E.g filter item (Price).
7. Press the `Tab` key. Make sure the focus switches to the next filter item. E.g. "Color".
8. Press the `Space` key on the filter item "Color". The options should show. E.g. "Purple" and "Yellow"
9. Press the `Space` key. You should navigate through "Color" options.
10. Press the `Tab` key a few times until you focus the next filter item. E.g. "Material".
11. Press the `Tab` key a few more times until you focus the "Close" button again (that's when no filter option applied).

## Screenshots / Screen Captures (if appropriate)


## Checklist

* I have added translations for new strings, if necessary.
